<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>724</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Таксопарк</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tabOrders">
         <attribute name="title">
          <string>Заказы</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableView" name="tableViewOrders">
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="newOrderGroupBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Добавление заказа</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLineEdit" name="clientFullnameLineEdit">
               <property name="placeholderText">
                <string>ФИО клиента</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QComboBox" name="taxistComboBox">
               <property name="maxVisibleItems">
                <number>100</number>
               </property>
               <property name="placeholderText">
                <string>Работник (Таксист)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="placeStartLineEdit">
               <property name="placeholderText">
                <string>Откуда</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="priceSpinBox">
               <property name="minimum">
                <number>50</number>
               </property>
               <property name="maximum">
                <number>100000000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="placeEndLineEdit">
               <property name="placeholderText">
                <string>Куда</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="clientPhoneLineEdit">
               <property name="placeholderText">
                <string>Номер телефона клиента</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QPushButton" name="createOrderButton">
               <property name="text">
                <string>Добавить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="orderCancelButton">
              <property name="text">
               <string>Заказ отменен</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="orderCompleteButton">
              <property name="text">
               <string>Заказ выполнен</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateOrdersButton">
              <property name="text">
               <string>Обновить таблицу</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabTaxists">
         <attribute name="title">
          <string>Таксисты</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableView" name="tableViewTaxists"/>
          </item>
          <item>
           <widget class="QGroupBox" name="taxistsGroupBox">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>Добавление таксиста</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0" colspan="2">
              <widget class="QLineEdit" name="fullnameLineEdit">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>ФИО</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QLineEdit" name="nickLineEdit">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Позывной</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="4">
              <widget class="QLineEdit" name="passportLineEdit">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Паспортные данные</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="4">
              <widget class="QLineEdit" name="carLineEdit">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Данные машины</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Дата рождения</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="2">
              <widget class="QDateEdit" name="birthdayDateEdit"/>
             </item>
             <item row="3" column="3">
              <widget class="QPushButton" name="addNewTaxistButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Добавить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="deleteTaxistButton">
              <property name="text">
               <string>Удалить таксиста</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateTaxistsButton">
              <property name="text">
               <string>Обновить таблицу</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabReport">
         <attribute name="title">
          <string>Отчет</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QChartView" name="chartView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
